src/compute.rs

```diff
@@ -11,36 +11,76 @@ use crate::{
     util::hash_to_scalar,
 };
 
+#[allow(non_snake_case)]
+/// Compute the group commitment
+pub fn group_commitment(commitment_list: &[(Scalar, PublicNonce)]) -> Scalar {
+    let mut hasher = Sha256::new();
+    let prefix = "WSTS/group_commitment";
+
+    hasher.update(prefix.as_bytes());
+    for (id, public_nonce) in commitment_list {
+        hasher.update(id.to_bytes());
+        hasher.update(public_nonce.D.compress().as_bytes());
+        hasher.update(public_nonce.E.compress().as_bytes());
+    }
+
+    hash_to_scalar(&mut hasher)
+}
+
+#[allow(non_snake_case)]
+/// Compute the group commitment
+pub fn group_commitment_compressed(commitment_list: &[(Scalar, Compressed, Compressed)]) -> Scalar {
+    let mut hasher = Sha256::new();
+    let prefix = "WSTS/group_commitment";
+
+    hasher.update(prefix.as_bytes());
+    for (id, D, E) in commitment_list {
```

Comment:
Can we use different names for these? In general, even for mathy code, I tend to use the programming language style with full words/phrases for variables.

Reply:
In the `FROST` paper they are called `D, E` but in the `RFC` they are called `hiding_commitment, binding_commitment`, so I went with the `RFC` names.